curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-hash-table.english.md

```diff
@@ -32,9 +32,9 @@ tests:
   - text: The HashTable should have a lookup method.
     testString: assert((function() { var test = false; if (typeof HashTable !== 'undefined') { test = new HashTable() }; return ((typeof test.lookup) === 'function')})());
   - text: The add method should add key value pairs and the lookup method should return the values associated with a given key.
-    testString: assert((function() { var test = false; if (typeof HashTable !== 'undefined') { test = new HashTable() }; test.add('key', 'value'); return (test.lookup('key') === 'value')})());
+    testString: assert((function(){ var test = false; if (typeof HashTable !== 'undefined') { test = new HashTable(); }; test.add('key', 'value'); test.lookup = function(key){ var theHash = hash(key); if (this.collection.hasOwnProperty(theHash)) { return this.collection[theHash][key]; } return null }; var lookup = test.lookup('key'); test.lookup = null; return (lookup === 'value')})());
```

Comment:
Sorry for continuing to ask for changes, but I think this test needs to be reverted to what it was. Meaning use the users `lookup` method, because the test text here is "The add method should add key value pairs and the lookup method should return the values associated with a given key." And this is the only test on the challenge that verifies their method.

Reply:
I see, let's revert this change then.